### PR TITLE
Add ZMSCORE command support

### PR DESCRIPTION
**Pull request by vladvildanov**

### CODE REVIEW

**Comment by vladvildanov** [comment]
@chayim @sazzad16 @dvora-h

**Review by sazzad16 — APPROVED** [review]

File: src/Command/Redis/ZMSCORE.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+declare(strict_types=1);
+
+namespace Predis\Command\Redis;
+
+use Predis\Command\Command as RedisCommand;
+
+/**
+ * @link https://redis.io/commands/zmscore/
+ *
+ * @author Vladyslav Vildanov <vladyslav.vildanov@redis.com>
```
Comment (chayim) [inline src/Command/Redis/ZMSCORE.php]: Remove yourself as the author... Weirdly the library only has the one author.

File: src/Command/Redis/ZMSCORE.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ */
+
+declare(strict_types=1);
+
+namespace Predis\Command\Redis;
+
+use Predis\Command\Command as RedisCommand;
+
+/**
+ * @link https://redis.io/commands/zmscore/
```
Comment (chayim) [inline src/Command/Redis/ZMSCORE.php:8]: Maybe explain (add in comments) what zmscore is and does. The link is great -we do this in redis-py, and my dream is to always do this.

File: tests/Predis/Command/Redis/ZMSCORE_Test.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) Vladyslav Vildanov <vladyslav.vildanov@redis.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+declare(strict_types=1);
```
Comment (chayim) [inline tests/Predis/Command/Redis/ZMSCORE_Test.php]: remove because... (as discussed)

File: .gitignore [hunk 4]
```diff
@@ -4,3 +4,4 @@
 .phpunit.result.cache
 composer.lock
 phpunit.xml
+.idea/
```
Comment (chayim) [inline .gitignore]: It looks like people in this library only gitignore language specific items, not environment specific, so let's not contribute this.

**Pull request closed without merging**